.github/workflows/dev.yml

```diff
@@ -46,14 +47,24 @@ jobs:
         $File = (Get-ChildItem -Path . -Include 'EXILED.*.nupkg' -Recurse).Name
         Out-File -FilePath ${{ github.env }} -InputObject "PackageFile=$File" -Encoding utf-8 -Append
 
-    - name: Upload artifacts
-      uses: actions/upload-artifact@v2
-      with:
-        name: Build Results
-        path: bin/Release
-
     - name: Upload nuget package
       uses: actions/upload-artifact@v2
       with:
         name: ${{ env.PackageFile }}
         path: ${{ env.PackageFile }}
+        
+    - name: Get references
+      shell: pwsh
+      run: |
+        Invoke-WebRequest -Uri ${{ env.EXILED_DLL_ARCHIVER_URL }} -OutFile ${{ github.workspace }}/EXILED-DLL-Archiver.exe
+  
+    - name: Packaging results as tar.gz
+      shell: pwsh
+      run: ./packaging.ps1
+
+    - name: Upload artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: Exiled.tar.gz
```

Comment:
BuildResult ? for the name

Reply:
so it's make BuildResult.zip and inside our Exiled.tar.gz than we love :3

Reply:
We love having many different files.

Reply:
> We love having many different files.

we can't make that different ``Note that .zip is unavoidable, Github automatically zips artifacts.``

Reply:
> BuildResult ? for the name

It will end up being the same thing. See https://github.com/Misaka-ZeroTwo/EXILED/actions/runs/6880861314.

Reply:
Done